360-Videos-Metadata/360VideosMetadata.py

```diff
@@ -58,18 +58,26 @@
 # XML contents.
 rdf_prefix = " xmlns:rdf=\"http://www.w3.org/1999/02/22-rdf-syntax-ns#\" "
 
-spherical_xml = (
-    "<?xml version=\"1.0\"?>\n"
-    "<rdf:SphericalVideo"
-    " xmlns:rdf=\"http://www.w3.org/1999/02/22-rdf-syntax-ns#\""
-    " xmlns:GSpherical=\"http://ns.google.com/videos/1.0/spherical/\">\n"
-    "  <GSpherical:Spherical>true</GSpherical:Spherical>\n"
-    "  <GSpherical:Stitched>true</GSpherical:Stitched>\n"
-    "  <GSpherical:StitchingSoftware>Spherical Metadata Tool"
-    "</GSpherical:StitchingSoftware>\n"
-    "  <GSpherical:ProjectionType>equirectangular"
-    "</GSpherical:ProjectionType>\n"
-    "</rdf:SphericalVideo>")
+spherical_xml_header = \
+    """<?xml version=\"1.0\"?>\n
```

Comment:
just to make sure ... with triple quote strings you don't need to use \n's (the newlines in the code are interpreted as \n's)

another benefit is you can remove all the \" escaping

here + below for spherical_xml_contents